### PR TITLE
Included Sorting Option in Tables table

### PR DESCRIPTION
**Pull request by himanshusagar**
This is in reference to issue : https://github.com/opendatakit/opendatakit/issues/1377
User can sort tables in List View by "Name" as per his choice i.e in ascending and descending order. Tables has another field "TableId" which I skipped because sorting via TableId would have given same ordering.

ezgif com-video-to-gif

@jbeorse 
Please review.

### CODE REVIEW

**Comment by himanshusagar** [comment]
run tests

**Comment by wbrunette** [comment]
runtests

**Comment by jbeorse** [comment]
Hi @himanshusagar. Thanks for all your work on this so far. This is heading in the right direction. I just have a few notes.

You've made your changes against the "master" branch. We actually make all new changes on the "development" branch and merge them downstream to "demo" and then "master" as they become more stable and we release. Please move your changes to the "development" branch and make your pull request against that. This should be easy since the latest release was quite recent. I expect most or all of the files to be the same.

You've stored the property in SharedPreferences, which is generally a good instinct. However, this won't work for our purposes. In ODK 2.0 we store our properties in the PropertiesSingleton (see the [properties package in androidlibrary](https://github.com/opendatakit/androidlibrary/tree/development/androidlibrary_lib/src/main/java/org/opendatakit/properties)). This allows us to keep a different copy of the property for each different data management application that is used on our platform. For example, a Malaria application might sort their tables ascending while a Dengue application might sort descending, on the same device with the same Tables Android app. Please use the PropertiesSingleton to store your property, and in particular please use a device property rather than general property so that it is not cleared when syncing.

The wording you've given: "Name ASC" and "Name DSC" I think is a bit technical. Please change this to "Name (Alphabetical)" and "Name (Reverse Alphabetical)". 

Please remove the stray white space lines and formatting changes (such as line 352 in MainActivity).

File: tables_app/src/main/java/org/opendatakit/tables/activities/MainActivity.java [hunk 1]
```diff
@@ -319,6 +349,7 @@ public void swapScreens(ScreenType newScreenType) {
       throw new IllegalStateException("Unexpected default case");
     }
 
+
```
Comment (jbeorse) [inline tables_app/src/main/java/org/opendatakit/tables/activities/MainActivity.java]: Please remove stray white space like this.

File: tables_app/src/main/java/org/opendatakit/tables/fragments/TableManagerFragment.java [hunk 2]
```diff
@@ -211,53 +240,53 @@ public boolean onContextItemSelected(MenuItem item) {
...
-      AlertDialog.Builder alert = new AlertDialog.Builder(this.getActivity());
-      alert.setTitle(getString(R.string.confirm_remove_table))
-          .setMessage(getString(R.string.are_you_sure_remove_table, localizedDisplayName));
-      // OK Action => delete the table
-      alert.setPositiveButton(getString(R.string.yes), new DialogInterface.OnClickListener() {
-        public void onClick(DialogInterface dialog, int whichButton) {
-          // treat delete as a local removal -- not a server side deletion
-          DbHandle db = null;
-          try {
+      case R.id.table_manager_delete_table:
```
Comment (jbeorse) [inline tables_app/src/main/java/org/opendatakit/tables/fragments/TableManagerFragment.java]: Please undo formatting changes like this. They make it harder to see what code has actually changed, and harder to review history.

**Comment by himanshusagar** [comment]
I have generated new Pull Request(https://github.com/opendatakit/tables/pull/66) after making all changes. Hence, closing this one.

**Pull request closed without merging**